bin/auto-cpufreq

```diff
@@ -211,14 +214,40 @@ def main(config, daemon, debug, install, remove, live, log, monitor, stats, vers
                         auto_cpufreq_stats_file.close()
 
                     auto_cpufreq_stats_path.unlink()
-                # ToDo:
+                # ToDo: 
+                # {the following snippet also used in --update, update it there too(if required)}
```

Comment:
I don't understand why this comment is made here?

Reply:
the existing comment was done by someone(to add bluetooth_disable() later), since I used the same snippet, i added it inorder to make changes later if required.